doc/plugindev/gssapi.rst

```diff
@@ -90,10 +90,13 @@ formats are:
   order, followed by the mechanism OID, followed by the mechanism
   token.
 
-* For **gss_import_name**, the bytes 04 01, followed by a two-byte OID
-  length in big-endian order, followed by the mechanism OID, followed
-  by the bytes 06, followed by the OID length as a single byte,
-  followed by the mechanism OID, followed by the mechanism token.
+* For **gss_import_name**, the bytes 04 01, followed by two plus the
+  interposed mechanism OID length in big-endian order, followed by the
+  byte 06, followed by the interposed OID length as a single byte,
+  followed by the interposed mechanism OID itself, followed by the
+  four byte big-endian length of the mechanism token, followed by the
+  mechanism token itself.  input_name_type must also be set to
+  GSS_C_NT_EXPORT_NAME.
```

Comment:
I think the two commits affecting this file would be better combined.  All of the documentation changes will probably be backported to 1.15 so that they show up in krb5-latest on the web.

Before these changes, the text above the bullet points says "using the concatenated OID" (which pretty clearly refers to the "special mechanism OID which is the concatenation of...") and the bullet points say "mechanism OID".  Note that the bullet points are introduced as describing the mechglue token formats, not directly describing what an interposer mechanism should put there.

After these changes, there is an inconsistent mix of "interposer mechanism OID", "interposed OID", and "interposed mechanism OID", none of which have been defined.  I don't think this is clearer.

I would accept changing "mechanism OID" to "concatenated OID" in the bullet points, and changing the text above the bullet points.